### PR TITLE
Dako 189 fe create login/register page

### PR DESCRIPTION
**Pull request by KalleChen**
# PR Template
## Why do we need this PR?
* Rebuild login/register pages

## How did you address the issue?
* Redesign the login/register pages
* Change Routing for only /login and /register
* Add email and password check

<br />  

> ## PR readiness check list
> - [x] Did it pass the Standard JS check?
> - [x] Did you test this PR?  
> - [x] Did you explain the changes clearly?
> - [x] Does the PR title reference a Jira issue for the correct project (at the beginning of the title)?

### CODE REVIEW

File: website/src/logged_out/components/register_login/RegisterDialog.js [hunk 1]
```diff
@@ -76,105 +139,138 @@ function RegisterDialog (props) {
...
+                setEmailStatus(null)
                 setStatus(null)
               }
             }}
+            onBlur={checkEmail}
+            helperText={(() => {
+              if (emailStatus === 'invalidEmail') {
+                return '請輸電子郵件'
+              } else if (
+                registerState.message ===
```
Comment (amjltc295) [inline website/src/logged_out/components/register_login/RegisterDialog.js:214]: Checking with message is not a very good practice .. we may nee to re-design this API

File: website/src/logged_out/components/register_login/LoginDialog.js [hunk 2]
```diff
@@ -72,20 +61,48 @@ function LoginDialog (props) {
...
         loading={isLoading}
         onFormSubmit={e => {
           e.preventDefault()
           login()
         }}
-        hideBackdrop
-        headline='Login'
+        headline='Dako'
+        title='登入'
+        slogan='(Slogan)'
```
Comment (amjltc295) [inline website/src/logged_out/components/register_login/LoginDialog.js:78]: Please confirm with @Yuhsin to write something here or leave it blank

File: website/src/logged_out/components/register_login/RegisterDialog.js [hunk 3]
```diff
@@ -206,51 +303,64 @@ function RegisterDialog (props) {
                   }}
                 >
                   {' '}
-                  terms of service
+                  隱私權條款
```
Comment (amjltc295) [inline website/src/logged_out/components/register_login/RegisterDialog.js:306]: I cannot open this dialog

**Review by amjltc295 — CHANGES_REQUESTED** [review]
Good job! There are still some problems to fix. Also, could you change the icon for web tag to our logo?

File: website/src/logged_out/components/register_login/RegisterDialog.js (hunk 3 above)
Comment (KalleChen) [reply website/src/logged_out/components/register_login/RegisterDialog.js:299]: The design team said that this dialog hasn't been designed yet, so just keep this text link disabled.

File: website/src/logged_out/components/register_login/RegisterDialog.js (hunk 1 above)
Comment (KalleChen) [reply website/src/logged_out/components/register_login/RegisterDialog.js]: Just changed the default error detail for the register add show the message directly below the email field.

File: website/src/logged_out/components/register_login/LoginDialog.js (hunk 2 above)
Comment (KalleChen) [reply website/src/logged_out/components/register_login/LoginDialog.js]: Fixed

**Review by amjltc295 — APPROVED** [review]

**Pull request merged**